### PR TITLE
Create the form

### PR DESCRIPTION
**Pull request by derjubu**

### CODE REVIEW

File: neue-fische-form.html [hunk 1]
```diff
@@ -10,13 +10,138 @@
...
     <h1>Kontaktformular Neue Fische</h1>
         <h2>Der Weg in deine digitale Zukunft</h2>
-            <form action="" >
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
+            <form action="" class="form-contact">
```
Comment (PReichetanz) [inline neue-fische-form.html:13]: Wie ware es mit dem "aria-labeledby"-Attribut an dieser Stelle?

File: neue-fische-form.html [hunk 2]
```diff
@@ -10,13 +10,138 @@
...
+                    <label for="lastName">Nachname*</label>
+                    <input 
+                        type="text" 
+                        id="lastName" 
+                        name="lastName" 
+                        placeholder="z.B.: Müller, Meyer oder Schmidt"
+                        required>
+                </div>
+                <div>
+                    <label for="emailAdress">E-Mail-Adresse</label>
```
Comment (PReichetanz) [inline neue-fische-form.html:50]: There should be a * after "E-Mail-Adresse"

File: neue-fische-form.html [hunk 3]
```diff
@@ -10,13 +10,138 @@
...
+                </div>
+                <div>
+                    <span>Bevorzugt in der Region...*</span><br>
+                    <input 
+                        type="radio" 
+                        id="region-hh" 
+                        name="region" 
+                        value="hh" 
+                        required>
+                    <label for="salutation-hh">Hamburg</label>
```
Comment (PReichetanz) [inline neue-fische-form.html:104]: The for-attribute should mirror the id-attribute of the related <input>, shouldn't it?

File: neue-fische-form.html [hunk 4]
```diff
@@ -8,15 +8,140 @@
...
-            <form action="" >
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
-                <label for=""></label>
+            <form action="" class="form-contact" aria-labelledby="contact-neuefische">
+                <div>
+                    <span>Anrede*</span><br>
```
Comment (badabam) [inline neue-fische-form.html:15]: Look into `fieldset` and `legend` for grouping radio buttons

File: neue-fische-form.html [hunk 5]
```diff
@@ -8,15 +8,140 @@
...
+                        required>
+                    <label for="salutation-f">Frau</label>
+                </div>
+                <div>
+                    <label for="firstName">Vorname*</label>
+                    <input 
+                        type="text" 
+                        id="firstName" 
+                        name="firstName" 
+                        placeholder="z.B.: Jaqueline oder Hans-Jürgen"
```
Comment (badabam) [inline neue-fische-form.html:37]: I think one example for a first name should be enough to get the point.

File: neue-fische-form.html [hunk 6]
```diff
@@ -8,15 +8,140 @@
...
+                <div>
+                    <label for="lastName">Nachname*</label>
+                    <input 
+                        type="text" 
+                        id="lastName" 
+                        name="lastName" 
+                        placeholder="z.B.: Müller, Meyer oder Schmidt"
+                        required>
+                </div>
+                <div>
```
Comment (badabam) [inline neue-fische-form.html:49]: You might not need those divs – at least while writing pure html

File: neue-fische-form.html [hunk 7]
```diff
@@ -8,15 +8,140 @@
...
+                        placeholder="z.B. honey2000@gmx-net.de"
+                        required>
+                </div>
+                <div>
+                    <span>Ich interessiere mich für...*</span><br>
+                    <input 
+                        type="checkbox" 
+                        id="interest-web" 
+                        name="interest" 
+                        value="web"
```
Comment (badabam) [inline neue-fische-form.html:64]: Nice: same name for those checkboxes, but different values.

File: neue-fische-form.html [hunk 8]
```diff
@@ -8,15 +8,140 @@
...
+                    <input 
+                        type="checkbox" 
+                        id="interest-datascience" 
+                        name="interest" 
+                        value="datascience" 
+                        required>
+                    <label for="interest-datascience">Data Analytics</label>
+                </div>
+                <div>
+                    <span>Bevorzugt in der Region...*</span><br>
```
Comment (badabam) [inline neue-fische-form.html:97]: (in those cases again fieldset and legend)

**Review by badabam — COMMENTED** [review]
Nice work! I left some comments

**Pull request merged**